packages/components/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,85 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import * as RadixSlider from '@radix-ui/react-slider';
+import { type ForwardedRef, forwardRef } from 'react';
+
+import { type CommonAriaAttrs } from '#/utilities/types';
+
+import styles from './styles/slider.module.scss';
+
+export type SliderProps = {
+    id?: string;
+    // name?: string; // For some reason radix does nothing with this
```

Comment:
```suggestion
    name?: string;
```
No name get added in the DOM node?

Reply:
nothing in the DOM but according to the docs it should work as usual when included in a `<form>` so I will keep the property available regardless
<img width="416" alt="image" src="https://github.com/user-attachments/assets/7bba26f4-9f77-421d-987d-7fae0c8bb762">